src/PointEval/PointEvalHandler.jl

```diff
@@ -273,3 +273,42 @@ function get_func_interpolations(dh::DH, fieldname) where DH<:MixedDofHandler
     end
     return func_interpolations
 end
+
+# Iteration of PointEvalHandler
+struct PointIterator{PH<:PointEvalHandler}
+    ph::PH
+    current_idx::ScalarWrapper{Int}
+    nodes::Vector{Int}
+    coords::Vector{Vec{2,Float64}}
+end
+function PointIterator(ph::PointEvalHandler{G}) where {D,C,T,G<:Grid{D,C,T}}
+    # @assert isconcretetype(C) # Check mate MixedDofHandler
+    N = nnodes_per_cell(ph.grid, 1)
```

Comment:
This will still not work for mixed grids. The principle of the `MixedDofHandler` is that you should go one `FieldHandler` at a time, so perhaps the `PointIterator` should follow that similarily to the `CellIterator` and take in a cellset. Then it would iterate only over the points which belong to that cellset.

Reply:
I'm `resize!`-ing: https://github.com/Ferrite-FEM/Ferrite.jl/pull/425/files#diff-431dc655f343b9166f1e184fc15699963fa4dcbba696fee98fdcb4354e840d3eR303

Reply:
Okay, but how do you handle choosing the correct `PointValues` then? Up to the user?

Reply:
Yea I think so. You would have something like
```julia
for point in PointIterator(ph)
    point === nothing && continue # point was not found
    cid = cellid(point)
    if cid in triangleset
        reinit!(triangle_pv, point)
        function_gradient(pv, ...)
    elseif cid in quadset
        reinit!(quad_pv, point)
        function_gradient(pv, ...)
    end
end
```

Reply:
Since the dofhandlers currently are not involved at all in PointEvalHandler I think this is fine.